src/main/java/com/auth0/json/mgmt/guardian/TwilioFactorProvider.java

```diff
@@ -62,11 +69,15 @@ public String getFrom() {
      * Setter for the Twilio From number.
      *
      * @param from the from number to set.
+     * @throws IllegalArgumentException when both `from` and `messagingServiceSID` are set
      * @deprecated use the constructor instead
      */
     @Deprecated
     @JsonProperty("from")
-    public void setFrom(String from) {
+    public void setFrom(String from) throws IllegalArgumentException {
+        if (messagingServiceSID != null) {
```

Comment:
If the user is removing the `from` i.e. setting `from=null` then the exception will be thrown anyway. The check should be: 

```java
if (from!=null && messagingServiceSID!=null){
    //.. 
}
```